docs/development/contributing_guide.md

```diff
@@ -169,6 +169,21 @@ To increase the log level for the tests, set `SYNAPSE_TEST_LOG_LEVEL`:
 SYNAPSE_TEST_LOG_LEVEL=DEBUG trial tests
 ```
 
+By default, tests will use an in-memory SQLite database for test data. For additional
+help with debugging, one can use an on-disk SQLite database file instead, in order to
+review database state during and after running tests. This can be adjusted using the
+`SYNAPSE_TEST_PERSIST_SQLITE_DB` environment variable:
+
+```sh
+SYNAPSE_TEST_PERSIST_SQLITE_DB=test.db trial tests
```

Comment:
We don't actually read the value of this variable, just the fact that it's been set. The path`test.db` is hard-coded for now.

```suggestion
SYNAPSE_TEST_PERSIST_SQLITE_DB=1 trial tests
```